### PR TITLE
Include process of unfavouriting client

### LINKED ISSUE #201: Incorrect name of `File` menu
image

Hi Jiet @hsiaojietng , i think you meant to say "File" instead of "files" 

This is in `DeveloperGuide.md`

https://github.com/AY2122S2-CS2103-W16-4/tp/blob/89dede2c36fe526a2cbc1315b93849b742942e18/docs/DeveloperGuide.md?plain=1#L524 

https://github.com/AY2122S2-CS2103-W16-4/tp/blob/89dede2c36fe526a2cbc1315b93849b742942e18/docs/DeveloperGuide.md?plain=1#L537

image

### PR DESCRIPTION
**Pull request by hsiaojietng**
Add documentation of unfavouriting a client with favouriting a client.

Closes #201

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -241,14 +245,14 @@ Step 5: The set of `UserImage` is then converted into an `ArrayList` and the fir
 ## Statistics feature/Window
 <img src="images/user-guide/Stats.png" height="400px">
 
-The `stats` opens a new `StatisticsWindow` that displays a pie chart with the data of the number of sellers & buyers in the 5 different regions, namely {North, South, East, West, Central}.
+The `stats` opens a new Statistics window that displays a pie chart with the data of the number of sellers & buyers in the 5 different regions, namely {North, South, East, West, Central}.
```
Comment (howardwhw2) [inline docs/DeveloperGuide.md:248]: Maybe adding the word command to become "The `stats` command opens" will be better?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -213,17 +213,20 @@ Examples:
 **Note**:
 - Images associated with the client are not deleted and remains in the OS file system.
 
-### Favourite a client: `favourite`
+### Favourite/Unfavourite a client: `favourite`
```
Comment (howardwhw2) [inline docs/UserGuide.md:216]: You might have to update the table of contents as well

**Review by howardwhw2 — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 2 above)
Comment (hsiaojietng) [reply docs/UserGuide.md:216]: Thanks for the reminder!

**Review by cindyangXQ — APPROVED** [review]
LGTM

**Pull request merged**